peachjam/views/legal_instrument.py

```diff
@@ -10,10 +10,9 @@ class LegalInstrumentListView(FilteredDocumentListView):
     model = LegalInstrument
     template_name = "peachjam/legal_instrument_list.html"
     navbar_link = "legal_instruments"
-    queryset = LegalInstrument.objects.prefetch_related("author", "nature")
 
     def get_queryset(self):
-        queryset = super(LegalInstrumentListView, self).get_queryset()
+        queryset = LegalInstrument.objects.prefetch_related("author", "nature")
```

Comment:
I think this breaks filtering on the page.

Reply:
@longhotsummer Thanks for pointing it out. I've seen the breaking change.